### PR TITLE
Notes comp

### PR DESCRIPTION
**Pull request by srishtigautam21**
Made the homePage for notes app with basic routing
Files to check:
HomePge.jsx
homePage.css
Sidebar.jsx
sidebar.css

### CODE REVIEW

File: src/component/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { NavLink } from "react-router-dom";
+import "./sidebar.css";
+
+const Sidebar = () => {
+  const activeStyle = ({ isActive }) => {
+    return isActive
+      ? "active-link md-pd aside-heading  "
+      : "inactive-link md-pd aside-heading  ";
+  };
+
```
Comment (itsmebipul00) [inline src/component/Sidebar/Sidebar.jsx:10]: Appreciation : Nice that you did this with active link

**Review by itsmebipul00 — APPROVED** [review]
Awsome work! Good to merge

File: src/component/Sidebar/Sidebar.jsx (hunk 1 above)
Comment (srishtigautam21) [reply src/component/Sidebar/Sidebar.jsx:10]: thanks bipul

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+
+import { LandingPage, HomePage, Archive, Labels, Trash } from "./pages";
 
 function App() {
+  const { pathname } = useLocation();
+  console.log(pathname);
   return (
     <>
-      <LandingPage />
+      {/* {pathname !== "/" && <Navbar />} */}
```
Comment (zoef-yt) [inline src/App.js:11]: Remove commented code.

File: src/component/Navbar/navbar.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
+.navbar {
+  grid-area: navbar;
+  box-shadow: 2px 2px 10px var(--BOX-SHADOW-LIGHTER);
+}
+.nav-padding {
+  padding: 0 2rem 0 2rem;
+}
+.icons-alignment {
+  /* margin: 2.5rem 3rem 0 0; */
```
Comment (zoef-yt) [inline src/component/Navbar/navbar.css]: remove commented code.

File: src/component/Navbar/navbar.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  max-width: 25rem;
+  width: 100%;
+}
+.link:hover {
+  cursor: pointer;
+  color: var(--PRIMARY-HOVER);
+}
+/* .lg-margin-top {
+  margin-top: 4.5rem;
+} */
```
Comment (zoef-yt) [inline src/component/Navbar/navbar.css]: remove commented code.

File: src/component/Sidebar/sidebar.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
+.sidebar {
+  grid-area: sidebar;
+  /* border-right: 1px solid #000; */
```
Comment (zoef-yt) [inline src/component/Sidebar/sidebar.css]: remove commented code.

File: src/component/Sidebar/sidebar.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  margin-top: 2rem;
+}
+.md-pd {
+  padding: 1.2rem 0.5rem 1.2rem 1.5rem;
+}
+.aside-heading {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 2rem;
+  color: #000;
```
Comment (zoef-yt) [inline src/component/Sidebar/sidebar.css]: Use var for colours

File: src/component/Sidebar/sidebar.css [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  cursor: pointer;
+}
+
+.active-link {
+  background-color: var(--darkerofseconadary);
+  border-top-right-radius: 1.5rem;
+  border-bottom-right-radius: 1.5rem;
+}
+.inactive-link:hover {
+  background-color: rgb(222, 222, 222);
```
Comment (zoef-yt) [inline src/component/Sidebar/sidebar.css:27]: Use var for colours

File: src/pages/HomePage/HomePage.jsx [hunk 8]
```diff
@@ -0,0 +1,19 @@
+// import { Navbar } from "../../component/Navbar/Navbar";
+// import { Sidebar } from "../../component/Sidebar/Sidebar";
```
Comment (zoef-yt) [inline src/pages/HomePage/HomePage.jsx]: remove commented code.

**Review by zoef-yt — APPROVED** [review]
You are good to merge with the changes. Good work Srishti.

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

File: src/App.js (hunk 2 above)
Comment (srishtigautam21) [reply src/App.js:11]: i have kept it for  future use to refactor my code in future

**Pull request merged**